src/selectors-api/selectors-api.json

```diff
@@ -0,0 +1,67 @@
+{
+    "BADGE": "badge",
+    "BREADCRUMBS": "breadcrumbs",
+    "BUTTON": {
+        "BUTTON": "btn",
```

Comment:
It looks fine, but it seems kinda weird to have `BUTTON` and then `BUTTON` again. Maybe call it `root` or `main` instead? 
@ianmcburnie thoughts?

Reply:
Yeah, let's go with `MAIN`.